module_utils/kafka_manager.py

```diff
@@ -183,6 +190,49 @@ def delete_topics(self, topics, timeout=None):
                     )
                 )
 
+    def get_coordinator_for_consumer_group(self, consumer_group=None):
+
+        # Identify which broker is coordinating this consumer group
+        response = self.send_request_and_get_response(
+                 GroupCoordinatorRequest_v0(consumer_group)
+                 )
+        return response.coordinator_id
```

Comment:
Can you check `error_code` on your requests? https://github.com/dpkp/kafka-python/blob/f0a57a6a20a3049dc43fbf7ad9eab9635bd2c0b0/kafka/protocol/commit.py#L216

Reply:
done